docs-src/content/functions/crypto.yml

```diff
@@ -158,6 +158,57 @@ funcs:
         +hIz6+EUt/Db51awO7iCuRly5L4TZ+CnMAsIbtUOqsqwSQDtv0AclAuogmCst75o
         aztsmrD79OXXnhUlURI=
         -----END PUBLIC KEY-----
+  - name: crypto.Ed25519GenerateKey
+    experimental: true
+    released: v4.0.0
+    description: |
+      Generate a new Ed25519 Private Key and output in
+      PEM-encoded PKCS#8 ASN.1 DER form.
+    pipeline: true
```

Comment:
because this function has no arguments, `pipeline` makes no sense (and breaks rendering with a cryptic error!)

```suggestion
```